### DISCUSSION TITLE
I can't use "git lfs install" command

### DISCUSSION THREAD

**Discussion by NicerWritter27** [git-lfs/git-lfs#4924] [Q&A]
After I installed "GitHub LFS", I go to my command line (cmd), and typed "git lfs install" command, but this happens:

![image](https://user-images.githubusercontent.com/92959844/160741113-27313a68-57ef-43ab-a432-67d97d9c2249.png)

So I tried to open the "git-lfs.exe" but it said:

![image](https://user-images.githubusercontent.com/92959844/160741262-9450d0f8-084e-42d0-b7d3-64c32e6c364b.png)

 So I opened the "git-lfs.exe" from my cmd:

![image](https://user-images.githubusercontent.com/92959844/160741472-7961412b-8754-412a-bb53-f6ae816987f4.png)

But IT WON'T WORK!!!!

WHAT IS WRONG WITH THIS!!!

I AM ANNOYED RIGHT NOW

PLEASE! ANYONE! TELL ME WHYYYY!

**Comment by chrisd8088** [comment]
Hey, I'm sorry you're having trouble.  The key error message is the one from the first and third screenshots, namely, you don't have Git installed, or at least it's not in your `PATH` and therefore can't be found when you run `git`.

Since you're using Windows, I might suggest installing the Git for Windows project, which comes with both Git and Git LFS.  Their Web page is here: https://gitforwindows.org/

**Reply by NicerWritter27** [reply]
what "PATH"? can you tell me where should I place "git-lfs.exe"?

**Reply by chrisd8088** [reply]
`PATH` is what's called an "environment variable"; most operating systems have such a concept.  It contains a series of directory paths and tells the operating system (Windows, in this case) where to look for a file to run.  So when you type "git", Windows searches through all the directories listed in your `PATH`, trying to find `git.exe`.  But you either don't have Git installed at all, or don't have it in a directory that's listed in your `PATH` environment variable.

You can read more about that in the Microsoft [documentation](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/path#remarks) and there are many other online sites with information about this.

Technically, you can put `git-lfs.exe` wherever you like if you know how to setup your `PATH` correctly, but Git LFS won't work without `git.exe` and all of Git's other support files.  You absolutely need Git and all of its dependencies installed before Git LFS will work at all.  That's why installing Git using the Git for Windows project makes sense, because it will place Git in a common directory and make sure that directory is in your `PATH`, and it also conveniently happens to bundle Git LFS along with Git, so you don't need to worry about the installation details.

It's maybe useful to explain that Git is an open source project (see https://git-scm.com/) and Git LFS is an extension to that project (see https://git-lfs.github.com/).  Both are open source projects that are independent of any particular company like GitLib or GitHub or Atlassian; such companies just provide Git-based hosting services and also offer support for hosting large files using Git LFS.

If you're using one of these companies' services, they each have documentation on how to use Git LFS (and, of course, Git too), such as:
- https://www.atlassian.com/git/tutorials/git-lfs
- https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage
